packages/console/src/components/RadioGroup/index.module.scss

```diff
@@ -107,6 +130,7 @@
       box-shadow: var(--shadow-2);
     }
 
+    /* stylelint-disable-next-line no-descending-specificity */
```

Comment:
Can we reorg the styling structure instead of simply disabling the rules? Is there any specific reason?

Reply:
These styles are in different catalogs, but the linter cannot recognize that.
```
.compact {
  // styles
}

.plain {
  // styles
}

.card {
  // styles
}
```
cc @charIeszhao

Reply:
Updated.